src/setup-elixir.js

```diff
@@ -1,65 +1,97 @@
 const core = require('@actions/core')
 const {exec} = require('@actions/exec')
-const {installElixir, installOTP} = require('./installer')
+const installer = require('./installer')
 const path = require('path')
 const semver = require('semver')
 const https = require('https')
 const {
-  fstat,
+  _fstat,
   promises: {readFile},
 } = require('fs')
 
-main().catch(err => {
+main().catch((err) => {
   core.setFailed(err.message)
 })
 
 async function main() {
-  checkPlatform()
```

Comment:
Same operations as performed, as per testing. I just made some stuff optional and moved other stuff around to follow the Erlang > Elixir > rebar3 exposed previously